ingestion-edge/tests/integration/publish/test_healthy.py

```diff
@@ -0,0 +1,121 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+
+from .helpers import IntegrationTest
+from ingestion_edge.config import METADATA_HEADERS
+from itertools import cycle, chain
+from typing import Dict, Optional
+import gzip
+import pytest
+
+# avoid haveing long strings in parametrized test
+# names by outsourcing the actual values
+DATA = {
+    b"GZIPPED": gzip.compress(b"data"),
```

Comment:
i had at one time intended to use something closer to a default map with a default value equal to the key and this was a holdover from that. i changed it to be string keys, because it doesn't matter.